src/lib/queries/getAccountsTotals.ts

```diff
@@ -33,7 +33,10 @@ export default async function getAccountsTotals(
   const totals: { [guid: string]: Money } = {};
 
   rows.forEach(row => {
-    totals[row.accountId] = new Money(row.total, accountsMap[row.accountId].commodity.mnemonic);
+    totals[row.accountId] = new Money(
+      accountsMap[row.accountId].type === 'INCOME' ? Math.abs(row.total) : row.total,
```

Comment:
for INCOME accounts, make it a positive quantity so we avoid having to do Math.abs in each visualization